tests/typecheck/managers/querysets/test_bulk_update.yml

```
@@ -0,0 +1,146 @@
+-   case: bulk_update_valid_fields
+    installed_apps:
+        - myapp
+    main: |
+        from myapp.models import Article, Author, Category
+
+        # Valid single field updates
+        articles = Article.objects.all()
+        Article.objects.bulk_update(articles, ["title"])
+        Article.objects.bulk_update(articles, ["content"])
+        Article.objects.bulk_update(articles, ["published"])
+
+        # Valid multiple field updates
+        Article.objects.bulk_update(articles, ["title", "content"])
```

```suggestion
        Article.objects.bulk_update(articles, ("title", "content"))
```

just for the variaty.